test/js/specs/happy.js

```diff
@@ -587,4 +587,19 @@ describe('Happy Paths', options, ({driver, pageObjects}) => {
       crossDeviceIntro.verifyMessages(crossDeviceIntroCopy)
     })
   })
+
+  describe('MODAL VIEW', async () => {
+
+    it('should be able to open, close and open again a modal view', async () => {
+      driver.get(localhostUrl + `?useModal=true`)
+      const welcomeCopy = welcome.copy()
+      welcome.clickOnOpenModalButton()
+      welcome.verifyTitle(welcomeCopy)
+      driver.sleep(500)
+      welcome.clickOnCloseModalButton()
+      driver.sleep(500)
+      welcome.clickOnOpenModalButton()
+      welcome.verifyTitle(welcomeCopy)
```

Comment:
I think this test is fine as it is, but maybe there should be another test for the modal to close on `Esc` key press?

Reply:
I will add it in the separate ticket.